lib/archive.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fsPromises = require('fs').promises;
+
+const NODE_ENV = process.env.NODE_ENV;
+const { mArchiveFile, mgArchiveFile, MgProcess } = require('../config.js');
+
+class MArchive {
```

Comment:
I do not recall what 'M' and 'Mg' represent. I suggest considering changing the names to names with a meaning that is immediately clear to the reader.

Reply:
'M' is the name of this tool, but I should definitely add a note here clarifying 👍